release-notes/SupportedFeatures-v1.0.0.md

```diff
@@ -34,28 +32,27 @@ Supported features
-|              |UPNIdentity    |[:warning:](#known_issues)          |[:warning:](#known_issues)          |[:warning:](#known_issues)                  | [:warning:](#known_issues)          |
+|              |SPNIdentity    |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)                  | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |
+|              |UPNIdentity    |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)                  | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |
 |Client types  |ChannelFactory\<T\>    |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |ChannelBase\<T\>       |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |Extensibility |IClientMessageInspector|:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |IClientMessageFormatter|:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|Management    |ETW Tracing            |[:no_entry_sign:](#known_issues)      |:white_check_mark:                  |[:no_entry_sign:](#known_issues)          |[:no_entry_sign:](#known_issues)     |          
+|Management    |ETW Tracing            |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)      |:white_check_mark:                  |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)     |          
 
 
 #### Platforms
 &ensp;&ensp;For more details about what UWP is please see this [dotnet Glossary doc](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/glossary.md).
 
-&ensp;&ensp;Linux OSs includes centos, debian, fedora, linux, opensuse, rhel and ubuntu.
+&ensp;&ensp;Linux OSs includes Red Hat Enterprise Linux, CentOS, Debian, Fedora, Linux Mint, OpenSUSE, Oracle Linux and Ubuntu.
 
 &ensp;&ensp;For supported versions of each platform and other details please see the [.NET Core 1.0.0 Release Notes](https://github.com/dotnet/core/blob/master/release-notes/1.0/Release-Notes-1.0.0.md).
```

Comment:
The link was changed to https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.0.md